pkgs/development/python-modules/sopel/default.nix

```
@@ -29,20 +30,26 @@ buildPythonPackage rec {
     pyenchant
     pygeoip
     pytz
+    sqlalchemy
     xmltodict
   ];
 
   # remove once https://github.com/sopel-irc/sopel/pull/1653 lands
   postPatch = ''
     substituteInPlace requirements.txt \
-      --replace "praw<6.0.0" "praw<7.0.0"
+      --replace "praw>=4.0.0,<6.0.0" "praw>=4.0.0,<8.0.0"
```

```suggestion
      --replace "praw>=4.0.0,<6.0.0" "praw"
```
Could try not locking version and just have failed tests indicate that it doesn't work.